include/openthread/platform/radio.h

```diff
@@ -203,20 +203,16 @@ typedef struct otRadioFrame
         struct
         {
             /**
-             * The timestamp when the frame was received (milliseconds).
-             * Applicable/Required only when raw-link-api feature (`OPENTHREAD_ENABLE_RAW_LINK_API`) is enabled.
+             * The timestamp when the frame was received in microseconds.
              *
-             */
-            uint32_t mMsec;
-
-            /**
-             * The timestamp when the frame was received (microseconds, the offset to mMsec).
-             * Applicable/Required only when raw-link-api feature (`OPENTHREAD_ENABLE_RAW_LINK_API`) is enabled.
+             * The value SHALL be the time when the SFD was received when TIME_SYNC or CSL is enabled,
+             * Otherwise it MAY be set to the time when rx was done.
```

Comment:
How about: "Otherwise, it MUST be set to the time when the MAC frame was fully received."?

Reply:
Shall we allow setting this to SFD time when CSL and TIME_SYNC is not enabled? How about:
"The value SHALL be the time when the SFD was received when TIME_SYNC or CSL is enabled.
Otherwise, the time when the MAC frame was fully received is also acceptable."?